### PR TITLE
adding storyteller specific db config prerequisites on the worker

### PR DESCRIPTION
**Pull request by JoeNunnelley**

### CODE REVIEW

**Comment by JoeNunnelley** [comment]
retest this please

File: recipes/worker-pg96.rb [hunk 1]
```diff
@@ -26,6 +26,8 @@
 
 cookbook_file "#{user_home}/postgres-setup.sh" do
   source 'postgres-setup.sh'
+  owner node['worker_user']
```
Comment (michaelb990) [inline recipes/worker-pg96.rb]: why is this necessary?

Comment (JoeNunnelley) [reply recipes/worker-pg96.rb]: hm guess its not. I'll remove.

**Comment by JoeNunnelley** [comment]
@michaelb990  I attempted to reduce the redundancy in the postgres-setup.sh script and handle the case where the db is already initialized so we don't get a bunch of annoying error messages.  If this really bugs you I can revert it.

**Comment by michaelb990** [comment]
why would we get error messages? this should only run once on worker startup. 

the way you're doing the file marking whether or not the DB is initialized should not use a relative path.

i still prefer the redundancy because i would prefer to keep this script as simple as possible. what is the issue with this specific case? it's not like we're writing complex logic over and over again where it would make sense to factor it out. this kind of redundancy actually improves readability in my opinion.

File: recipes/worker-pg94.rb [hunk 2]
```diff
@@ -17,15 +17,7 @@
...
 
-# add & run postgres setup script.
-# TODO fix the code below so that it doesn't need the || true block and doesn't fail building in Jenkins
-bash 'setup postgres' do
-  code "#{user_home}/postgres-setup.sh || true"
-  action :nothing
-end
-
-cookbook_file "#{user_home}/postgres-setup.sh" do
+cookbook_file '/etc/profile.d/postgres-setup.sh' do
```
Comment (michaelb990) [inline recipes/worker-pg94.rb]: This should not be in `profile.d`.

File: recipes/worker-pg96.rb [hunk 3]
```diff
@@ -17,15 +17,7 @@
...
 
-# add & run postgres setup script.
-# TODO fix the code below so that it doesn't need the || true block and doesn't fail building in Jenkins
-bash 'setup postgres' do
-  code "#{user_home}/postgres-setup.sh || true"
-  action :nothing
-end
-
-cookbook_file "#{user_home}/postgres-setup.sh" do
+cookbook_file '/etc/profile.d/postgres-setup.sh' do
```
Comment (michaelb990) [inline recipes/worker-pg96.rb]: Same for this.

File: recipes/_common.rb [hunk 4]
```diff
@@ -8,6 +8,12 @@
   EOH
 end
 
+bash 'set locale' do
```
Comment (michaelb990) [inline recipes/_common.rb]: What is this for?

**Comment by JoeNunnelley** [comment]
retest this please

**Comment by JoeNunnelley** [comment]
retest this please

**Comment by JoeNunnelley** [comment]
@michaelb990  I believe I've addressed your comments.
